inbm/dispatcher-agent/dispatcher/ota_parser.py

```diff
@@ -176,8 +176,9 @@ def parse(self, resource: Dict, kwargs: Dict, parsed: XmlHandler) -> Dict[str, A
             logger.info("Config Params not passed correctly"
                         " in manifest, rejected update")
             raise XmlException
-        kwargs.update({'signature': self._signature, 'config_params': config_params,
+        kwargs.update({'config_params': config_params,
```

Comment:
Ensure that the order of updates to `kwargs` does not affect the outcome, as the order of dictionary keys is not guaranteed in Python versions before 3.7. [medium]